src/display/display_utils.js

```diff
@@ -428,6 +448,19 @@ class DummyStatTimer {
   }
 }
 
+function isValidFetchUrl(url, baseUrl) {
```

Comment:
Could you add a unit test for this new utility function?

Reply:
That's a fair point, will fix this.